src/helm/benchmark/metrics/vision_language/image_metrics.py

```diff
@@ -309,6 +312,8 @@ def compute_ssim(self, generated_image: np.ndarray, reference_image: np.ndarray)
         return ssim(generated_image, reference_image)
 
     def compute_edit_sim(self, completion: str, reference: str) -> float:
+        from helm.benchmark.metrics.copyright_metrics import _edit_similarity
```

Comment:
I think this import should stay at the top or otherwise we should do this for all the metrics